orangecontrib/text/vectorization/document_embedder.py

```diff
@@ -0,0 +1,185 @@
+"""This module contains classes used for embedding documents
+into a vector space.
+"""
+import zlib
+import base64
+import json
+import sys
+import warnings
+from typing import Tuple, Any, Optional
+import numpy as np
+
+from Orange.misc.server_embedder import ServerEmbedderCommunicator
+from orangecontrib.text import Corpus
+
+
+AGGREGATORS = ['mean', 'sum', 'max', 'min']
```

Comment:
I would prefer to have these in camel case. Similar to these: https://github.com/biolab/orange3-timeseries/blob/cc459739d8ee901bc93b2c624c08f1c0c9ba9adc/orangecontrib/timeseries/agg_funcs.py#L113